src/ontology/uberon.Makefile

```diff
@@ -1306,6 +1347,19 @@ $(COMPONENTSDIR)/hra_depiction_3d_images.owl: $(TMPDIR)/hra_depiction_3d_images.
 	$(ROBOT) merge -i $< annotate --ontology-iri $(ONTBASE)/$@ --output $@
 
 
+# The mappings.owl component contains cross-references to foreign
+# ontologies that provide their own mappings as a SSSOM set. This
+# component ensures those mappings are visible to Uberon editors and
+# users.
+$(COMPONENTSDIR)/mappings.owl: $(SRC) $(EXTERNAL_SSSOM_SETS) $(TMPDIR)/plugins/sssom.jar
```

Comment:
@anitacaron lets remember this pattern for the future: having the plugin path as dependency, and wget ing it the same way as a regular dependency!